sql/src/test/java/org/apache/druid/sql/calcite/BaseCalciteQueryTest.java

```diff
@@ -1019,6 +1047,31 @@ void validate(int row, int column, ValueType type, Object expectedCell, Object r
               (Double) resultCell,
               eps
           );
+        } else if (expectedCell instanceof Object[] || expectedCell instanceof List) {
```

Comment:
what's the difference between this block and the above starting in line 995?

Reply:
They are the same, I will refactor it to a private method.